inc/packages/admin/info.php

```diff
@@ -151,22 +175,22 @@ function render( MetadataDocument $doc, string $tab, string $section ) {
 					'<div id="section-%s" class="section" style="display: %s;">%s</div>',
 					esc_attr( $section_id ),
 					( $section_id === $section ) ? 'block' : 'none',
-					$prepared
+					esc_attr( $prepared )
```

Comment:
This will sanitize the HTML content as if it was attributes.

Reply:
Should we run it back through `sanitize_html()`?

Reply:
Either that or `esc_html()` I guess.

Reply:
Or not `esc_html()` :joy: What was I thinking saying that? It's bloody HTML!